src/telemetry/stats.h

```diff
@@ -69,15 +70,22 @@ typedef struct HyperStats
 	int64 uncompressed_row_count;
 } HyperStats;
 
+typedef struct CaggStats
+{
+	HyperStats hs;
```

Comment:
This name is unnecessarily terse.

Reply:
I actually think that PostgreSQL has unnecessarily terse names as well, but this is not the gist of my review. Pick whatever you feel is a good balance with regard to naming.

Reply:
Personally, I don't subscribe to "longer names is always better" and terse makes sense in structs when dealing with "base objects" in the inheritance sense. However, as you've seen I had longer and more descriptive names initially, so I am not against it. But given that Windows didn't accept the previous naming convention I had to come up with something new and opted for short and consistent. 

However, per your request, I reverted the naming and made and exception out of "hyper" instead.

Reply:
Keyword is "unnecessarily". :) No, longer names are not always better.